workshops/book-dash/2023-Nov-BookDash-10.md

```diff
@@ -0,0 +1,320 @@
+       * Chatted about governance - thanks Anne/Ale for facilitating these discussions!
+       * Read all the things about the inclusive event PRs started by Sophia! 
+   * Ask: Also opened up a new issue on the feedback process: if you have any resources/thoughts please weigh in: [https://github.com/the-turing-way/the-turing-way/issues/3361](https://github.com/the-turing-way/the-turing-way/issues/3361)
+
+   * Name: Alejandro
+       * Appreciations
+           * Amazing week with new and old faces in TTW book dash
+           * Nice chats with Maria N and Patricia from Metadocencia about the history of the Spanish translation team and trainings on satellite image technologies
+           * Calum for quickly capturing complex ideas into nice illustrations
+           * Anne F for joining the Scriberia session with Calum
+           * Accessibility funds of the Book Dash for the childcare support. Matilda had a great company and time with her babysitter, Marjana.
+       * Gave back (PRs reviewed - all them amazing contributions):
+           * Esther (Research Data Management): very interesting stuff about Data Repositories 
+           * Maria (TTW Spanish Translation Guidelines): thank you for bringing fresh ideas!
+           * Julien C (Open Hardware): very insighful for me and pertinent for some colleagues working on Open Hardware for Environmental monitoring and education 
+           * Richard (Ethical Licensing): this requires further reading, but it was so informative to learn about the fundamental freedoms of free/libre software and ! This is very relevant to a project I'm participating about Open Computer Vision.
+           * Notes:
+               * Alejandro's sixth week, very nice week hahaha late 
+
+### Videos:
+   * Share-out Session 1: [https://youtu.be/sRuVu\_6vO-E](https://youtu.be/sRuVu\_6vO-E)
+   * Share-out Session 2: [https://youtu.be/oqxj-AVtA-Y](https://youtu.be/oqxj-AVtA-Y)
+
+### New Illustrations by Scriberia for *The Turing Way* Community 
+   * Illustration files are posted on Zenodo: [https://zenodo.org/record/3332807](https://zenodo.org/record/3332807).
+   * Particular set from this Book Dash is available at: [https://zenodo.org/records/10556824](https://zenodo.org/records/10556824)
+   * Please reuse them and cite as *The Turing Way Community, \& Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. *[https://doi.org/10.5281/zenodo.3332807Feedback](*https://doi.org/10.5281/zenodo.3332807*Feedback)
+
+## Pluses 
+At the end of the event, we asked our participants to share feedback anonymously in the “Pluses and Deltas” HackMD.
+We greatly appreciate the work that our attendees have accomplished in the project during this short event and thank them for their feedback, a few of which have been highlighted below.
+
+Copy from: [https://annuel2.framapad.org/p/ttw-bookdash-nov2023-feedback](https://annuel2.framapad.org/p/ttw-bookdash-nov2023-feedback)
```

Comment:
Should we have this link here of the original framapad?

Reply:
No, it should be removed